dist/api_data.py

```diff
@@ -217,6 +217,8 @@ def __ge__(self, other):
             the bucket indicating the location for this table'''),
         Config('bucket_prefix', '', r'''
             the unique bucket prefix for this table'''),
+        Config('bucket_region', '', r'''
+            bucket region string identifier, represents the bucket's location'''),
```

Comment:
The "bucket" string was intended to incorporate the region (and anything else) into it by the user/caller. Is there any way the callers can just do that? Or does the S3 extension have to extract the region out specifically?

Reply:
Hmm, having region as part of bucket name will actually make things simpler for the tiered storage and let extensions decide how they want to use that input. That makes sense. It should not be hard to make that change.